src/Tooltip/TooltipNext/docs/index.story.js

```diff
@@ -27,6 +27,7 @@ import { Layout, Cell } from '../../../Layout';
 import { baseScope } from '../../../../stories/utils/LiveCodeExample';
 import usage from './Usage.md';
 import * as examples from './examples';
+import Readme from '!raw-loader!./README.TESTKIT.md';
```

Comment:
raw-loader for markdown is not needed
```suggestion
import Readme from './README.TESTKIT.md';
```